changelog/23.0/23.0.0/summary.md

```diff
@@ -13,6 +16,18 @@
         - [CLI Flags](#flags-vttablet)
         - [Managed MySQL configuration defaults to caching-sha2-password](#mysql-caching-sha2-password)
 
+## <a id="major-changes"/>Major Changes</a>
+
+### <a id="vtorc"/>VTOrc</a>
+
+#### <a id="vtorc-dynamic-ers-disabled"/>Dynamic control of `EmergencyReparentShard`-based recoveries</a>
+
+**Note: disabling `EmergencyReparentShard`-based recoveries introduces availability risks; please use with extreme caution! If you rely on this functionality often, for example in automation, this may be signs of an anti-pattern. If so, please open an issue to discuss supporting your use case natively in VTOrc.**
+
+The new `vtctldclient` RPC `SetVtorcEmergencyReparent` was introduced to allow VTOrc recoveries involving `EmergencyReparentShard` actions to be disabled on a per-keyspace and/or per-shard basis. Previous to this version, disabling ERS-based recoveries was only possible globally/per-VTOrc-instance. VTOrc will now consider this keyspace/shard-level setting that is refreshed from the topo on each recovery.
+
+To provide observability of keyspace/shards with ERS-based VTOrc recoveries disabled, the label `ErsDisabled` was added to the `TabletsWatchedByShard` metric. This metric label can be used to create alerting to ensure ERS-based recoveries are not disabled for an undesired period of time.
```

Comment:
For other related metrics, we deprecated names that use `Ers` and expanded them to be more descriptive. It may be a good idea to follow that pattern for the new metric.

Reply:
@deepthi that's a good point. These acronyms might not be obvious to people that are less familiar. Will update 👍

Reply:
Updated! A label like `emergency_reparent_shard_disabled` seemed to big, so I moved this to a new by-keyspace/shard metric named `EmergencyReparentShardDisabled` and updated the changelog 👍